pkg/vm/stack_item.go

```diff
@@ -240,3 +240,17 @@ func toMapKey(key StackItem) interface{} {
 		panic("wrong key type")
 	}
 }
+
+// fromMapKey converts map key to StackItem.
+func fromMapKey(key interface{}) StackItem {
```

Comment:
Do we really need it? It seems like a `makeStackItem()` duplicate. I don't think you should care about wrong key types here, because you have this `key` coming from the map on the stack and supposedly you have already checked it when you've created it.

Reply:
I'd prefer to keep them separate. We can make `fromMapKey` a wrapper of `makeStackItem` but semantically they are different, it is just a coincidence they do the same thing. Separate function will also be useful in case we will do performance optimizations or refactoring.

Regarding your second point it is also used for a user input. It will be absolutely necessary to validate keys before then.

Reply:
> Regarding your second point it is also used for a user input

But where? I only see single use case in `KEYS`. I'm not convinced, really.

Reply:
Yeah, I was thinking about `toMapKey`.

Reply:
And the last question is --- do we really need this wrapper?

Reply:
We don't _need_ it, but imo code was less error-prone with it.
Removed.